app/share.js

```diff
@@ -58,9 +60,7 @@ class Root extends React.Component {
 	}
 
 	init = async() => {
-		if (isIOS) {
-			await RNUserDefaults.setName('group.ios.chat.rocket');
-		}
+		await RNUserDefaults.setName(IDENTIFIER);
```

Comment:
I think something similar should be done here: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/sagas/deepLinking.js#L37

Reply:
Good catch! Adding...